syncstorage-mysql/migrations/2018-08-28-010336_init/up.sql

```diff
@@ -1,69 +1,55 @@
 -- /*!40100 DEFAULT CHARACTER SET latin1 */
-
 -- DROP TABLE IF EXISTS `bso`;
 -- XXX: bsov1, etc
 CREATE TABLE IF NOT EXISTS `bso`(
-    `user_id` INT                           NOT NULL,
-    `collection_id` INT                     NOT NULL,
-    `id` VARCHAR(64)                        NOT NULL,
-
+    `user_id` INT NOT NULL,
```

Comment:
nit: did the IDE reformat this? I've tried to keep this styled after https://www.sqlstyle.guide/#create-syntax

Reply:
Yeah, it must have gotten auto-formatted.